grafana-plugin/src/utils/hooks.tsx

```diff
@@ -25,25 +25,12 @@ export function useOnClickOutside(ref, handler) {
   }, [ref, handler]);
 }
 
-export function usePrevious(value: any) {
```

Comment:
Although not in use **right now**, I wouldn't remove this one. This hook can anytime be used within a functional component.

Reply:
added back 👍